.github/workflows/build-pr-artifacts.yml

```diff
@@ -52,14 +52,15 @@ jobs:
         with:
           context: .
           file: Dockerfile
+          target: development
           load: true
           tags: |
             ${{ env.REPO_NAME }}:${{ env.TAG_NAME }}
 
       - name: Run Tests
         run: |
           docker run ${{ env.REPO_NAME }}:${{ env.TAG_NAME }} npm run test:ci
-
+          docker run ${{ env.REPO_NAME }}:${{ env.TAG_NAME }} npm run test:integration
```

Comment:
We cannot directly run integration tests. It needs more setup (see test.yml).

Reply:
This is not the user transform integration tests these are `service.integration.test.js` which is running test e2e with supertest

Reply:
Got it, but why are we explicitly ignoring those tests in the config file?

Reply:
Because we want to have separate build and deployment for user-transform images

Reply:
I don't think we'd discard that functionality from DT images anytime soon. Migration has to happen for all the users (including open-source).
So, we still have to run those tests on the DT images. Try to find a different way of doing it.